package.json

```diff
@@ -48,6 +48,7 @@
   "scripts": {
     "postinstall": "node patches/apply-patches.js",
     "prebuild": "rimraf build/ build-bin/",
+    "pretarball": "ts-node --transpile-only ../../automation/run.ts sign:binaries",
```

Comment:
The path here needs to be relative to where oclif will be when running the BuildTar step (which will always be `<root>/tmp/balena`

Reply:
Moving this into this step instead of patching the library was oclif's mantainer suggestion for our workflow: https://github.com/oclif/oclif/issues/1331